package.json

```diff
@@ -93,6 +93,17 @@
     "webpack": "5.38.1"
   },
   "resolutions": {
-    "graphql": "15.5.1"
+    "graphql": "15.5.1",
+    "immer": "9.0.6",
+    "node-fetch": "2.6.7",
+    "shelljs": "0.8.5",
+    "follow-redirects": "1.14.7",
+    "axios": "0.21.2",
+    "tar": "6.1.9",
+    "prismjs": "1.25.0",
+    "trim-newlines": "3.0.1",
+    "glob-parent": "5.1.2",
+    "trim": "0.0.3",
+    "elliptic": "6.5.4"
```

Comment:
These are sub-dependencies of some other packages we use in our projects. We should update the top-level packages instead.

To help you with it, here is the dependency tree for each of the package you've listed here:

```text
$ for p in immer node-fetch shelljs follow-redirects axios tar prismjs trim-newlines glob-parent trim elliptic ; do \
    echo "------------------------------------------ $p -------------------------------------------" ; \
    yarn why -R $p ; \
  done > /tmp/why.log
```

<details>
<summary>
Content of /tmp/why.log
</summary>

```text
------------------------------------------ immer -------------------------------------------
├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber
│  └─ @storybook/react@npm:6.2.9 [546c7] (via npm:6.2.9 [546c7])
│     ├─ @storybook/core@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │  └─ @storybook/core-server@npm:6.2.9 [cdace] (via npm:6.2.9 [cdace])
│     │     └─ @storybook/builder-webpack4@npm:6.2.9 [d14e6] (via npm:6.2.9 [d14e6])
│     │        └─ react-dev-utils@npm:11.0.4 [17c6b] (via npm:^11.0.3 [17c6b])
│     │           └─ immer@npm:8.0.1 (via npm:8.0.1)
│     └─ react-dev-utils@npm:11.0.4 [57ca0] (via npm:^11.0.3 [57ca0])
│        └─ immer@npm:8.0.1 (via npm:8.0.1)
│
├─ @labelflow/web@workspace:typescript/web
│  ├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber [08ba8] (via workspace:typescript/react-openlayers-fiber [08ba8])
│  └─ @storybook/react@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│     ├─ @storybook/core@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│     │  └─ @storybook/core-server@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│     │     └─ @storybook/builder-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│     │        └─ react-dev-utils@npm:11.0.4 [17c6b] (via npm:^11.0.3 [17c6b])
│     └─ react-dev-utils@npm:11.0.4 [57ca0] (via npm:^11.0.3 [57ca0])
│
└─ labelflow@workspace:.
   ├─ @storybook/builder-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  └─ react-dev-utils@npm:11.0.4 [e3246] (via npm:^11.0.3 [e3246])
   │     └─ immer@npm:8.0.1 (via npm:8.0.1)
   └─ @storybook/react@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
      ├─ @storybook/core@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
      │  └─ @storybook/core-server@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
      │     └─ @storybook/builder-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
      │        └─ react-dev-utils@npm:11.0.4 [17c6b] (via npm:^11.0.3 [17c6b])
      └─ react-dev-utils@npm:11.0.4 [57ca0] (via npm:^11.0.3 [57ca0])
------------------------------------------ node-fetch -------------------------------------------
├─ @labelflow/common-resolvers@workspace:typescript/common-resolvers
│  └─ isomorphic-fetch@npm:3.0.0 (via npm:3.0.0)
│     └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│
├─ @labelflow/db@workspace:typescript/db
│  ├─ @labelflow/common-resolvers@workspace:typescript/common-resolvers (via workspace:typescript/common-resolvers)
│  ├─ @supabase/supabase-js@npm:1.21.0 (via npm:1.21.0)
│  │  ├─ @supabase/gotrue-js@npm:1.17.0 (via npm:^1.17.0)
│  │  │  └─ cross-fetch@npm:3.1.4 (via npm:^3.0.6)
│  │  │     └─ node-fetch@npm:2.6.1 (via npm:2.6.1)
│  │  ├─ @supabase/postgrest-js@npm:0.33.3 (via npm:^0.33.0)
│  │  │  └─ cross-fetch@npm:3.1.4 (via npm:^3.0.6)
│  │  └─ @supabase/storage-js@npm:1.4.1 (via npm:^1.4.0)
│  │     └─ cross-fetch@npm:3.1.4 (via npm:^3.1.0)
│  ├─ isomorphic-fetch@npm:3.0.0 (via npm:3.0.0)
│  └─ apollo-server@npm:3.5.0 [99545] (via npm:3.5.0 [99545])
│     ├─ apollo-server-core@npm:3.5.0 [874ae] (via npm:^3.5.0 [874ae])
│     │  ├─ apollo-datasource@npm:3.3.0 (via npm:^3.3.0)
│     │  │  └─ apollo-server-env@npm:4.2.0 (via npm:^4.2.0)
│     │  │     └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│     │  ├─ apollo-server-env@npm:4.2.0 (via npm:^4.2.0)
│     │  ├─ apollo-server-plugin-base@npm:3.4.0 [30c31] (via npm:^3.4.0 [30c31])
│     │  │  └─ apollo-server-types@npm:3.4.0 [30c31] (via npm:^3.4.0 [30c31])
│     │  │     └─ apollo-server-env@npm:4.2.0 (via npm:^4.2.0)
│     │  └─ apollo-server-types@npm:3.4.0 [30c31] (via npm:^3.4.0 [30c31])
│     └─ apollo-server-express@npm:3.5.0 [874ae] (via npm:^3.5.0 [874ae])
│        ├─ apollo-server-core@npm:3.5.0 [874ae] (via npm:^3.5.0 [874ae])
│        └─ apollo-server-types@npm:3.4.0 [30c31] (via npm:^3.4.0 [30c31])
│
├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber
│  └─ @storybook/react@npm:6.2.9 [546c7] (via npm:6.2.9 [546c7])
│     └─ @storybook/core@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│        └─ @storybook/core-server@npm:6.2.9 [cdace] (via npm:6.2.9 [cdace])
│           └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│
├─ @labelflow/scripts@workspace:typescript/scripts
│  └─ @supabase/supabase-js@npm:1.21.0 (via npm:1.21.0)
│
├─ @labelflow/update-license@workspace:typescript/release
│  └─ auto-changelog@npm:2.3.0 (via npm:2.3.0)
│     └─ node-fetch@npm:2.6.1 (via npm:^2.6.0)
│
├─ @labelflow/web@workspace:typescript/web
│  ├─ @labelflow/common-resolvers@workspace:typescript/common-resolvers (via workspace:typescript/common-resolvers)
│  ├─ @labelflow/db@workspace:typescript/db (via workspace:typescript/db)
│  ├─ @supabase/supabase-js@npm:1.21.0 (via npm:1.21.0)
│  ├─ auto-changelog@npm:2.3.0 (via npm:2.3.0)
│  ├─ mjml@npm:4.10.3 (via npm:4.10.3)
│  │  ├─ mjml-cli@npm:4.10.3 (via npm:4.10.3)
│  │  │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │  │     └─ juice@npm:7.0.0 (via npm:^7.0.0)
│  │  │        └─ web-resource-inliner@npm:5.0.0 (via npm:^5.0.0)
│  │  │           └─ node-fetch@npm:2.6.1 (via npm:^2.6.0)
│  │  ├─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │  └─ mjml-preset-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-accordion@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-body@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-button@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-carousel@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-column@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-divider@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-group@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head-attributes@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head-breakpoint@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head-font@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head-html-attributes@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head-preview@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head-style@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head-title@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-head@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-hero@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-image@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-navbar@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-raw@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-section@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-social@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-spacer@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-table@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     ├─ mjml-text@npm:4.10.3 (via npm:4.10.3)
│  │     │  └─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │     └─ mjml-wrapper@npm:4.10.3 (via npm:4.10.3)
│  │        ├─ mjml-core@npm:4.10.3 (via npm:4.10.3)
│  │        └─ mjml-section@npm:4.10.3 (via npm:4.10.3)
│  ├─ release-it@npm:14.11.5 (via npm:14.11.5)
│  │  └─ @octokit/rest@npm:18.9.0 (via npm:18.9.0)
│  │     └─ @octokit/core@npm:3.5.1 (via npm:^3.5.0)
│  │        ├─ @octokit/graphql@npm:4.6.2 (via npm:^4.5.8)
│  │        │  └─ @octokit/request@npm:5.4.15 (via npm:^5.3.0)
│  │        │     └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│  │        └─ @octokit/request@npm:5.6.1 (via npm:^5.6.0)
│  │           └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│  ├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber [08ba8] (via workspace:typescript/react-openlayers-fiber [08ba8])
│  ├─ @sentry/nextjs@npm:6.12.0 [08ba8] (via npm:6.12.0 [08ba8])
│  │  └─ @sentry/webpack-plugin@npm:1.17.1 (via npm:1.17.1)
│  │     └─ @sentry/cli@npm:1.68.0 (via npm:^1.68.0)
│  │        └─ node-fetch@npm:2.6.1 (via npm:^2.6.0)
│  ├─ @storybook/react@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│  │  └─ @storybook/core@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │     └─ @storybook/core-server@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│  │        ├─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│  │        └─ @storybook/manager-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│  │           └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│  ├─ apollo@npm:2.33.9 [08ba8] (via npm:2.33.9 [08ba8])
│  │  ├─ apollo-env@npm:0.10.1 (via npm:^0.10.1)
│  │  │  └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│  │  ├─ apollo-codegen-core@npm:0.40.7 [20d49] (via npm:^0.40.7 [20d49])
│  │  │  ├─ apollo-env@npm:0.10.1 (via npm:^0.10.1)
│  │  │  └─ apollo-language-server@npm:1.26.7 [20d49] (via npm:^1.26.7 [20d49])
│  │  │     ├─ apollo-datasource@npm:0.9.0 (via npm:^0.9.0)
│  │  │     │  └─ apollo-server-env@npm:3.1.0 (via npm:^3.1.0)
│  │  │     │     └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│  │  │     └─ apollo-env@npm:0.10.1 (via npm:^0.10.1)
│  │  ├─ apollo-codegen-flow@npm:0.38.7 [20d49] (via npm:^0.38.7 [20d49])
│  │  │  └─ apollo-codegen-core@npm:0.40.7 [20d49] (via npm:^0.40.7 [20d49])
│  │  ├─ apollo-codegen-scala@npm:0.39.7 [20d49] (via npm:^0.39.7 [20d49])
│  │  │  └─ apollo-codegen-core@npm:0.40.7 [20d49] (via npm:^0.40.7 [20d49])
│  │  ├─ apollo-codegen-swift@npm:0.40.7 [20d49] (via npm:^0.40.7 [20d49])
│  │  │  └─ apollo-codegen-core@npm:0.40.7 [20d49] (via npm:^0.40.7 [20d49])
│  │  ├─ apollo-codegen-typescript@npm:0.40.7 [20d49] (via npm:^0.40.7 [20d49])
│  │  │  └─ apollo-codegen-core@npm:0.40.7 [20d49] (via npm:^0.40.7 [20d49])
│  │  └─ apollo-language-server@npm:1.26.7 [20d49] (via npm:^1.26.7 [20d49])
│  ├─ graphiql@npm:1.5.16 [08ba8] (via npm:1.5.16 [08ba8])
│  │  ├─ codemirror-graphql@npm:1.2.11 [57211] (via npm:^1.2.11 [57211])
│  │  │  └─ graphql-language-service@npm:4.1.4 [57211] (via npm:^4.1.4 [57211])
│  │  │     ├─ graphql-language-service-interface@npm:2.10.2 [079e8] (via npm:^2.10.2 [079e8])
│  │  │     │  ├─ graphql-config@npm:4.1.0 [241ad] (via npm:^4.1.0 [241ad])
│  │  │     │  │  └─ @graphql-tools/url-loader@npm:7.7.0 [14d95] (via npm:^7.4.2 [14d95])
│  │  │     │  │     ├─ cross-undici-fetch@npm:0.1.4 (via npm:^0.1.4)
│  │  │     │  │     │  └─ node-fetch@npm:2.6.6 (via npm:^2.6.5)
│  │  │     │  │     └─ sync-fetch@npm:0.3.1 (via npm:^0.3.1)
│  │  │     │  │        └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
│  │  │     │  ├─ graphql-language-service-parser@npm:1.10.4 [079e8] (via npm:^1.10.4 [079e8])
│  │  │     │  │  └─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     │  │     └─ graphql-config@npm:4.1.0 [241ad] (via npm:^4.1.0 [241ad])
│  │  │     │  ├─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     │  └─ graphql-language-service-utils@npm:2.7.1 [079e8] (via npm:^2.7.1 [079e8])
│  │  │     │     └─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     ├─ graphql-language-service-parser@npm:1.10.4 [079e8] (via npm:^1.10.4 [079e8])
│  │  │     ├─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     └─ graphql-language-service-utils@npm:2.7.1 [079e8] (via npm:^2.7.1 [079e8])
│  │  └─ graphql-language-service@npm:4.1.4 [57211] (via npm:^4.1.4 [57211])
│  └─ next@npm:12.0.7 [08ba8] (via npm:12.0.7 [08ba8])
│     └─ node-fetch@npm:2.6.1 (via npm:2.6.1)
│
├─ infrastructure@workspace:typescript/infrastructure
│  └─ @pulumi/kubernetes@npm:3.7.2 (via npm:3.7.2)
│     └─ node-fetch@npm:2.6.5 (via npm:^2.3.0)
│
└─ labelflow@workspace:.
   ├─ auto-changelog@npm:2.3.0 (via npm:2.3.0)
   ├─ chromatic@npm:5.8.2 (via npm:5.8.2)
   │  ├─ @actions/github@npm:4.0.0 (via npm:^4.0.0)
   │  │  └─ @octokit/core@npm:3.4.0 (via npm:^3.0.0)
   │  │     ├─ @octokit/graphql@npm:4.6.2 (via npm:^4.5.8)
   │  │     └─ @octokit/request@npm:5.4.15 (via npm:^5.4.12)
   │  └─ node-fetch@npm:2.6.1 (via npm:^2.6.0)
   ├─ jest-fetch-mock@npm:3.0.3 (via npm:3.0.3)
   │  └─ cross-fetch@npm:3.1.4 (via npm:^3.0.4)
   ├─ release-it@npm:14.11.5 (via npm:14.11.5)
   ├─ @graphql-codegen/cli@npm:1.21.4 [3f179] (via npm:1.21.4 [3f179])
   │  ├─ @graphql-tools/apollo-engine-loader@npm:6.2.5 [8b930] (via npm:^6 [8b930])
   │  │  └─ cross-fetch@npm:3.0.6 (via npm:3.0.6)
   │  │     └─ node-fetch@npm:2.6.1 (via npm:2.6.1)
   │  ├─ @graphql-tools/github-loader@npm:6.2.5 [8b930] (via npm:^6 [8b930])
   │  │  └─ cross-fetch@npm:3.0.6 (via npm:3.0.6)
   │  ├─ @graphql-tools/prisma-loader@npm:6.3.0 [8b930] (via npm:^6 [8b930])
   │  │  ├─ isomorphic-fetch@npm:3.0.0 (via npm:^3.0.0)
   │  │  ├─ @graphql-tools/url-loader@npm:6.10.1 [8b930] (via npm:^6 [8b930])
   │  │  │  ├─ cross-fetch@npm:3.1.4 (via npm:3.1.4)
   │  │  │  └─ sync-fetch@npm:0.3.0 (via npm:0.3.0)
   │  │  │     └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
   │  │  └─ graphql-request@npm:3.4.0 [2ce59] (via npm:^3.3.0 [2ce59])
   │  │     └─ cross-fetch@npm:3.1.4 (via npm:^3.0.6)
   │  ├─ @graphql-tools/url-loader@npm:6.10.1 [8b930] (via npm:^6 [8b930])
   │  └─ graphql-config@npm:3.3.0 [8b930] (via npm:^3.2.0 [8b930])
   │     └─ @graphql-tools/url-loader@npm:6.10.1 [8b930] (via npm:^6 [8b930])
   ├─ @graphql-codegen/typescript-resolvers@npm:1.19.1 [3f179] (via npm:1.19.1 [3f179])
   │  ├─ @graphql-codegen/typescript@npm:1.22.0 [3f179] (via npm:1.22.0 [3f179])
   │  │  └─ @graphql-codegen/visitor-plugin-common@npm:1.20.0 [5f785] (via npm:^1.20.0 [5f785])
   │  │     └─ @graphql-tools/relay-operation-optimizer@npm:6.3.0 [064a5] (via npm:^6 [064a5])
   │  │        └─ relay-compiler@npm:10.1.0 [5834c] (via npm:10.1.0 [5834c])
   │  │           ├─ fbjs@npm:3.0.0 (via npm:^3.0.0)
   │  │           │  └─ cross-fetch@npm:3.1.4 (via npm:^3.0.4)
   │  │           └─ relay-runtime@npm:10.1.0 (via npm:10.1.0)
   │  │              └─ fbjs@npm:3.0.0 (via npm:^3.0.0)
   │  └─ @graphql-codegen/visitor-plugin-common@npm:1.20.0 [5f785] (via npm:^1.20.0 [5f785])
   ├─ @graphql-codegen/typescript@npm:1.22.0 [3f179] (via npm:1.22.0 [3f179])
   ├─ @storybook/manager-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
   └─ @storybook/react@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
      └─ @storybook/core@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
         └─ @storybook/core-server@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
            ├─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
            └─ @storybook/manager-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
               └─ node-fetch@npm:2.6.1 (via npm:^2.6.1)
------------------------------------------ shelljs -------------------------------------------
├─ @labelflow/web@workspace:typescript/web
│  ├─ release-it@npm:14.11.5 (via npm:14.11.5)
│  │  └─ shelljs@npm:0.8.4 (via npm:0.8.4)
│  └─ apollo@npm:2.33.9 [08ba8] (via npm:2.33.9 [08ba8])
│     └─ git-rev-sync@npm:3.0.1 (via npm:3.0.1)
│        └─ shelljs@npm:0.8.4 (via npm:0.8.4)
│
├─ infrastructure@workspace:typescript/infrastructure
│  └─ @kubernetes/client-node@npm:0.15.1 (via npm:0.15.1)
│     └─ shelljs@npm:0.8.4 (via npm:^0.8.4)
│
└─ labelflow@workspace:.
   └─ release-it@npm:14.11.5 (via npm:14.11.5)
------------------------------------------ follow-redirects -------------------------------------------
├─ @labelflow/web@workspace:typescript/web
│  └─ bundlewatch@npm:0.3.2 (via npm:0.3.2)
│     └─ axios@npm:0.21.1 (via npm:^0.21.1)
│        └─ follow-redirects@npm:1.14.1 [d192f] (via npm:^1.10.0 [d192f])
│
└─ labelflow@workspace:.
   └─ chromatic@npm:5.8.2 (via npm:5.8.2)
      └─ @chromaui/localtunnel@npm:2.0.3 (via npm:^2.0.2)
         └─ axios@npm:0.21.1 (via npm:0.21.1)
------------------------------------------ axios -------------------------------------------
├─ @labelflow/web@workspace:typescript/web
│  └─ bundlewatch@npm:0.3.2 (via npm:0.3.2)
│     └─ axios@npm:0.21.1 (via npm:^0.21.1)
│
└─ labelflow@workspace:.
   └─ chromatic@npm:5.8.2 (via npm:5.8.2)
      └─ @chromaui/localtunnel@npm:2.0.3 (via npm:^2.0.2)
         └─ axios@npm:0.21.1 (via npm:0.21.1)
------------------------------------------ tar -------------------------------------------
├─ @labelflow/db@workspace:typescript/db
│  ├─ @supabase/supabase-js@npm:1.21.0 (via npm:1.21.0)
│  │  └─ @supabase/realtime-js@npm:1.1.3 (via npm:^1.1.1)
│  │     └─ websocket@npm:1.0.34 (via npm:^1.0.34)
│  │        ├─ bufferutil@npm:4.0.3 (via npm:^4.0.1)
│  │        │  └─ node-gyp@npm:8.0.0 (via npm:latest)
│  │        │     ├─ make-fetch-happen@npm:8.0.14 (via npm:^8.0.14)
│  │        │     │  └─ cacache@npm:15.0.6 (via npm:^15.0.5)
│  │        │     │     └─ tar@npm:6.1.0 (via npm:^6.0.2)
│  │        │     └─ tar@npm:6.1.0 (via npm:^6.1.0)
│  │        └─ utf-8-validate@npm:5.0.5 (via npm:^5.0.2)
│  │           └─ node-gyp@npm:8.0.0 (via npm:latest)
│  └─ nodemon@npm:2.0.12 (via npm:2.0.12)
│     └─ chokidar@npm:3.5.2 (via npm:^3.2.2)
│        └─ fsevents@patch:fsevents@npm%3A2.3.2#~builtin<compat/fsevents>::version=2.3.2&hash=1cc4b2 (via patch:fsevents@~2.3.2#~builtin<compat/fsevents>)
│           └─ node-gyp@npm:8.0.0 (via npm:latest)
│
├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber
│  └─ @storybook/react@npm:6.2.9 [546c7] (via npm:6.2.9 [546c7])
│     ├─ @storybook/core@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │  └─ @storybook/core-server@npm:6.2.9 [cdace] (via npm:6.2.9 [cdace])
│     │     ├─ @storybook/builder-webpack4@npm:6.2.9 [d14e6] (via npm:6.2.9 [d14e6])
│     │     │  ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │     │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
│     │     │  │  │  └─ chokidar@npm:3.5.1 (via npm:^3.4.2)
│     │     │  │  │     └─ fsevents@patch:fsevents@npm%3A2.3.2#~builtin<compat/fsevents>::version=2.3.2&hash=1cc4b2 (via patch:fsevents@~2.3.1#~builtin<compat/fsevents>)
│     │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     │     │  │     ├─ node-libs-browser@npm:2.2.1 (via npm:^2.2.1)
│     │     │  │     │  └─ crypto-browserify@npm:3.12.0 (via npm:^3.11.0)
│     │     │  │     │     ├─ browserify-cipher@npm:1.0.1 (via npm:^1.0.0)
│     │     │  │     │     │  ├─ browserify-aes@npm:1.2.0 (via npm:^1.0.4)
│     │     │  │     │     │  │  └─ evp_bytestokey@npm:1.0.3 (via npm:^1.0.3)
│     │     │  │     │     │  │     └─ node-gyp@npm:8.0.0 (via npm:latest)
│     │     │  │     │     │  └─ evp_bytestokey@npm:1.0.3 (via npm:^1.0.0)
│     │     │  │     │     ├─ browserify-sign@npm:4.2.1 (via npm:^4.0.0)
│     │     │  │     │     │  └─ parse-asn1@npm:5.1.6 (via npm:^5.1.5)
│     │     │  │     │     │     ├─ browserify-aes@npm:1.2.0 (via npm:^1.0.0)
│     │     │  │     │     │     └─ evp_bytestokey@npm:1.0.3 (via npm:^1.0.0)
│     │     │  │     │     └─ public-encrypt@npm:4.0.3 (via npm:^4.0.0)
│     │     │  │     │        └─ parse-asn1@npm:5.1.6 (via npm:^5.0.0)
│     │     │  │     └─ watchpack@npm:1.7.5 (via npm:^1.7.4)
│     │     │  │        ├─ chokidar@npm:3.5.1 (via npm:^3.4.1)
│     │     │  │        └─ watchpack-chokidar2@npm:2.0.1 (via npm:^2.0.1)
│     │     │  │           └─ chokidar@npm:2.1.8 (via npm:^2.1.8)
│     │     │  │              └─ fsevents@patch:fsevents@npm%3A1.2.13#~builtin<compat/fsevents>::version=1.2.13&hash=1cc4b2 (via patch:fsevents@^1.2.7#~builtin<compat/fsevents>)
│     │     │  │                 └─ nan@npm:2.14.2 (via npm:^2.12.1)
│     │     │  │                    └─ node-gyp@npm:8.0.0 (via npm:latest)
│     │     │  ├─ terser-webpack-plugin@npm:3.1.0 [d14e6] (via npm:^3.1.0 [d14e6])
│     │     │  │  └─ cacache@npm:15.0.6 (via npm:^15.0.5)
│     │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     │     ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │     ├─ terser-webpack-plugin@npm:3.1.0 [d14e6] (via npm:^3.1.0 [d14e6])
│     │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│
├─ @labelflow/scripts@workspace:typescript/scripts
│  └─ @supabase/supabase-js@npm:1.21.0 (via npm:1.21.0)
│
├─ @labelflow/web@workspace:typescript/web
│  ├─ @labelflow/db@workspace:typescript/db (via workspace:typescript/db)
│  ├─ @supabase/supabase-js@npm:1.21.0 (via npm:1.21.0)
│  ├─ browser-builtins@npm:3.3.1 (via npm:3.3.1)
│  │  └─ crypto-browserify@npm:3.9.14 (via npm:~3.9.10)
│  │     ├─ browserify-aes@npm:1.2.0 (via npm:^1.0.0)
│  │     ├─ browserify-sign@npm:3.0.8 (via npm:^3.0.1)
│  │     │  └─ parse-asn1@npm:3.0.2 (via npm:^3.0.0)
│  │     │     ├─ browserify-aes@npm:1.2.0 (via npm:^1.0.0)
│  │     │     └─ evp_bytestokey@npm:1.0.3 (via npm:^1.0.0)
│  │     └─ public-encrypt@npm:2.0.1 (via npm:^2.0.0)
│  │        └─ parse-asn1@npm:3.0.2 (via npm:^3.0.0)
│  ├─ jest@npm:26.6.3 (via npm:26.6.3)
│  │  ├─ @jest/core@npm:26.6.3 (via npm:^26.6.3)
│  │  │  ├─ @jest/reporters@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  ├─ @jest/transform@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  │  └─ jest-haste-map@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  │     └─ fsevents@patch:fsevents@npm%3A2.3.2#~builtin<compat/fsevents>::version=2.3.2&hash=1cc4b2 (via patch:fsevents@^2.1.2#~builtin<compat/fsevents>)
│  │  │  │  └─ jest-haste-map@npm:26.6.2 (via npm:^26.6.2)
│  │  │  ├─ @jest/transform@npm:26.6.2 (via npm:^26.6.2)
│  │  │  ├─ jest-haste-map@npm:26.6.2 (via npm:^26.6.2)
│  │  │  ├─ jest-resolve-dependencies@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  └─ jest-snapshot@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │     └─ jest-haste-map@npm:26.6.2 (via npm:^26.6.2)
│  │  │  ├─ jest-runner@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  ├─ jest-haste-map@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  ├─ jest-runtime@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  │  ├─ @jest/transform@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  │  ├─ jest-haste-map@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  │  ├─ jest-snapshot@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  │  └─ jest-config@npm:26.6.3 [bf7a8] (via npm:^26.6.3 [bf7a8])
│  │  │  │  │     ├─ @jest/test-sequencer@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  │     │  ├─ jest-haste-map@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  │     │  ├─ jest-runner@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  │     │  └─ jest-runtime@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  │     ├─ jest-jasmine2@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  │     │  ├─ jest-runtime@npm:26.6.3 (via npm:^26.6.3)
│  │  │  │  │     │  └─ jest-snapshot@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  │     └─ babel-jest@npm:26.6.3 [caddf] (via npm:^26.6.3 [caddf])
│  │  │  │  │        └─ @jest/transform@npm:26.6.2 (via npm:^26.6.2)
│  │  │  │  └─ jest-config@npm:26.6.3 [bf7a8] (via npm:^26.6.3 [bf7a8])
│  │  │  ├─ jest-runtime@npm:26.6.3 (via npm:^26.6.3)
│  │  │  ├─ jest-snapshot@npm:26.6.2 (via npm:^26.6.2)
│  │  │  └─ jest-config@npm:26.6.3 [bf7a8] (via npm:^26.6.3 [bf7a8])
│  │  └─ jest-cli@npm:26.6.3 (via npm:^26.6.3)
│  │     ├─ @jest/core@npm:26.6.3 (via npm:^26.6.3)
│  │     └─ jest-config@npm:26.6.3 [bf7a8] (via npm:^26.6.3 [bf7a8])
│  ├─ mjml@npm:4.10.3 (via npm:4.10.3)
│  │  └─ mjml-cli@npm:4.10.3 (via npm:4.10.3)
│  │     └─ chokidar@npm:3.5.2 (via npm:^3.0.0)
│  ├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber [08ba8] (via workspace:typescript/react-openlayers-fiber [08ba8])
│  ├─ @storybook/react@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│  │  ├─ @storybook/core@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │  └─ @storybook/core-server@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│  │  │     ├─ @storybook/builder-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│  │  │     │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
│  │  │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     │  ├─ terser-webpack-plugin@npm:4.2.3 [17c6b] (via npm:^4.2.3 [17c6b])
│  │  │     │  │  └─ cacache@npm:15.0.6 (via npm:^15.0.5)
│  │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     ├─ @storybook/manager-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│  │  │     │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     │  ├─ terser-webpack-plugin@npm:4.2.3 [17c6b] (via npm:^4.2.3 [17c6b])
│  │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  ├─ next@npm:12.0.7 [08ba8] (via npm:12.0.7 [08ba8])
│  │  ├─ chokidar@npm:3.5.1 (via npm:3.5.1)
│  │  └─ crypto-browserify@npm:3.12.0 (via npm:3.12.0)
│  └─ node-polyfill-webpack-plugin@npm:1.1.2 [08ba8] (via npm:1.1.2 [08ba8])
│     └─ crypto-browserify@npm:3.12.0 (via npm:^3.12.0)
│
├─ infrastructure@workspace:typescript/infrastructure
│  └─ @kubernetes/client-node@npm:0.15.1 (via npm:0.15.1)
│     └─ tar@npm:6.1.0 (via npm:^6.0.2)
│
└─ labelflow@workspace:.
   ├─ jest@npm:26.6.3 (via npm:26.6.3)
   ├─ @graphql-codegen/cli@npm:1.21.4 [3f179] (via npm:1.21.4 [3f179])
   │  └─ chokidar@npm:3.5.1 (via npm:^3.4.3)
   ├─ @storybook/builder-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  ├─ @storybook/core-common@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
   │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  └─ fork-ts-checker-webpack-plugin@npm:6.2.9 [e3246] (via npm:^6.0.4 [e3246])
   │     └─ chokidar@npm:3.5.1 (via npm:^3.4.2)
   ├─ @storybook/manager-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  └─ @storybook/core-common@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   ├─ @storybook/react@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
   │  ├─ @storybook/core@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │  └─ @storybook/core-server@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
   │  │     ├─ @storybook/builder-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
   │  │     │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
   │  │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     │  ├─ terser-webpack-plugin@npm:4.2.3 [17c6b] (via npm:^4.2.3 [17c6b])
   │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     ├─ @storybook/manager-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
   │  │     │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     │  ├─ terser-webpack-plugin@npm:4.2.3 [17c6b] (via npm:^4.2.3 [17c6b])
   │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   └─ node-polyfill-webpack-plugin@npm:1.1.2 [3f179] (via npm:1.1.2 [3f179])
      └─ crypto-browserify@npm:3.12.0 (via npm:^3.12.0)
------------------------------------------ prismjs -------------------------------------------
├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber
│  └─ @storybook/react@npm:6.2.9 [546c7] (via npm:6.2.9 [546c7])
│     └─ @storybook/core@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│        ├─ @storybook/core-client@npm:6.2.9 [cdace] (via npm:6.2.9 [cdace])
│        │  └─ @storybook/ui@npm:6.2.9 [e3db7] (via npm:6.2.9 [e3db7])
│        │     └─ @storybook/components@npm:6.2.9 [cd127] (via npm:6.2.9 [cd127])
│        │        └─ react-syntax-highlighter@npm:13.5.3 [71330] (via npm:^13.5.3 [71330])
│        │           ├─ prismjs@npm:1.23.0 (via npm:^1.21.0)
│        │           └─ refractor@npm:3.3.1 (via npm:^3.1.0)
│        │              └─ prismjs@npm:1.23.0 (via npm:~1.23.0)
│        └─ @storybook/core-server@npm:6.2.9 [cdace] (via npm:6.2.9 [cdace])
│           ├─ @storybook/builder-webpack4@npm:6.2.9 [d14e6] (via npm:6.2.9 [d14e6])
│           │  ├─ @storybook/components@npm:6.2.9 [cd127] (via npm:6.2.9 [cd127])
│           │  └─ @storybook/ui@npm:6.2.9 [e3db7] (via npm:6.2.9 [e3db7])
│           ├─ @storybook/core-client@npm:6.2.9 [d14e6] (via npm:6.2.9 [d14e6])
│           │  └─ @storybook/ui@npm:6.2.9 [e3db7] (via npm:6.2.9 [e3db7])
│           └─ @storybook/ui@npm:6.2.9 [e3db7] (via npm:6.2.9 [e3db7])
│
├─ @labelflow/web@workspace:typescript/web
│  ├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber [08ba8] (via workspace:typescript/react-openlayers-fiber [08ba8])
│  ├─ @storybook/addon-actions@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│  │  └─ @storybook/components@npm:6.3.12 [6e07e] (via npm:6.3.12 [6e07e])
│  │     └─ react-syntax-highlighter@npm:13.5.3 [0e179] (via npm:^13.5.3 [0e179])
│  │        ├─ prismjs@npm:1.23.0 (via npm:^1.21.0)
│  │        └─ refractor@npm:3.3.1 (via npm:^3.1.0)
│  └─ @storybook/react@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│     └─ @storybook/core@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│        ├─ @storybook/core-client@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│        │  └─ @storybook/ui@npm:6.3.12 [dcbe6] (via npm:6.3.12 [dcbe6])
│        │     └─ @storybook/components@npm:6.3.12 [6e07e] (via npm:6.3.12 [6e07e])
│        └─ @storybook/core-server@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│           ├─ @storybook/builder-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│           │  ├─ @storybook/components@npm:6.3.12 [6e07e] (via npm:6.3.12 [6e07e])
│           │  └─ @storybook/ui@npm:6.3.12 [dcbe6] (via npm:6.3.12 [dcbe6])
│           ├─ @storybook/core-client@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│           │  └─ @storybook/ui@npm:6.3.12 [dcbe6] (via npm:6.3.12 [dcbe6])
│           └─ @storybook/manager-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│              ├─ @storybook/core-client@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│              └─ @storybook/ui@npm:6.3.12 [dcbe6] (via npm:6.3.12 [dcbe6])
│
└─ labelflow@workspace:.
   ├─ @storybook/addon-actions@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
   │  └─ @storybook/components@npm:6.3.12 [21d4a] (via npm:6.3.12 [21d4a])
   │     └─ react-syntax-highlighter@npm:13.5.3 [cef18] (via npm:^13.5.3 [cef18])
   │        ├─ prismjs@npm:1.23.0 (via npm:^1.21.0)
   │        └─ refractor@npm:3.3.1 (via npm:^3.1.0)
   ├─ @storybook/builder-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  └─ @storybook/components@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   │     └─ react-syntax-highlighter@npm:13.5.3 [cef18] (via npm:^13.5.3 [cef18])
   ├─ @storybook/manager-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  ├─ @storybook/core-client@npm:6.3.0-beta.15 [82ee4] (via npm:6.3.0-beta.15 [82ee4])
   │  │  └─ @storybook/ui@npm:6.3.0-beta.15 [82ee4] (via npm:6.3.0-beta.15 [82ee4])
   │  │     └─ @storybook/components@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   │  └─ @storybook/ui@npm:6.3.0-beta.15 [82ee4] (via npm:6.3.0-beta.15 [82ee4])
   └─ @storybook/react@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
      └─ @storybook/core@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
         ├─ @storybook/core-client@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
         │  └─ @storybook/ui@npm:6.3.12 [e9c0c] (via npm:6.3.12 [e9c0c])
         │     └─ @storybook/components@npm:6.3.12 [21d4a] (via npm:6.3.12 [21d4a])
         └─ @storybook/core-server@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
            ├─ @storybook/builder-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
            │  ├─ @storybook/components@npm:6.3.12 [21d4a] (via npm:6.3.12 [21d4a])
            │  └─ @storybook/ui@npm:6.3.12 [e9c0c] (via npm:6.3.12 [e9c0c])
            ├─ @storybook/core-client@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
            │  └─ @storybook/ui@npm:6.3.12 [e9c0c] (via npm:6.3.12 [e9c0c])
            └─ @storybook/manager-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
               ├─ @storybook/core-client@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
               └─ @storybook/ui@npm:6.3.12 [e9c0c] (via npm:6.3.12 [e9c0c])
------------------------------------------ trim-newlines -------------------------------------------
└─ labelflow@workspace:.
   └─ chromatic@npm:5.8.2 (via npm:5.8.2)
      └─ meow@npm:8.1.2 (via npm:^8.0.0)
         └─ trim-newlines@npm:3.0.0 (via npm:^3.0.0)
------------------------------------------ glob-parent -------------------------------------------
├─ @labelflow/db@workspace:typescript/db
│  ├─ nodemon@npm:2.0.12 (via npm:2.0.12)
│  │  └─ chokidar@npm:3.5.2 (via npm:^3.2.2)
│  │     └─ glob-parent@npm:5.1.2 (via npm:~5.1.2)
│  └─ @graphql-tools/load@npm:6.2.8 [8b930] (via npm:^6 [8b930])
│     └─ globby@npm:11.0.3 (via npm:11.0.3)
│        └─ fast-glob@npm:3.2.5 (via npm:^3.1.1)
│           └─ glob-parent@npm:5.1.2 (via npm:^5.1.0)
│
├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber
│  └─ @storybook/react@npm:6.2.9 [546c7] (via npm:6.2.9 [546c7])
│     ├─ @storybook/core@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │  └─ @storybook/core-server@npm:6.2.9 [cdace] (via npm:6.2.9 [cdace])
│     │     ├─ cpy@npm:8.1.2 (via npm:^8.1.1)
│     │     │  └─ globby@npm:9.2.0 (via npm:^9.2.0)
│     │     │     └─ fast-glob@npm:2.2.7 (via npm:^2.2.6)
│     │     │        └─ glob-parent@npm:3.1.0 (via npm:^3.1.0)
│     │     ├─ @storybook/builder-webpack4@npm:6.2.9 [d14e6] (via npm:6.2.9 [d14e6])
│     │     │  ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │     │  │  ├─ glob-base@npm:0.3.0 (via npm:^0.3.0)
│     │     │  │  │  └─ glob-parent@npm:2.0.0 (via npm:^2.0.0)
│     │     │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
│     │     │  │  │  └─ chokidar@npm:3.5.1 (via npm:^3.4.2)
│     │     │  │  │     └─ glob-parent@npm:5.1.2 (via npm:~5.1.0)
│     │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     │     │  │     └─ watchpack@npm:1.7.5 (via npm:^1.7.4)
│     │     │  │        ├─ chokidar@npm:3.5.1 (via npm:^3.4.1)
│     │     │  │        └─ watchpack-chokidar2@npm:2.0.1 (via npm:^2.0.1)
│     │     │  │           └─ chokidar@npm:2.1.8 (via npm:^2.1.8)
│     │     │  │              └─ glob-parent@npm:3.1.0 (via npm:^3.1.0)
│     │     │  ├─ react-dev-utils@npm:11.0.4 [17c6b] (via npm:^11.0.3 [17c6b])
│     │     │  │  └─ globby@npm:11.0.1 (via npm:11.0.1)
│     │     │  │     └─ fast-glob@npm:3.2.5 (via npm:^3.1.1)
│     │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     │     ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     ├─ react-dev-utils@npm:11.0.4 [57ca0] (via npm:^11.0.3 [57ca0])
│     │  └─ globby@npm:11.0.1 (via npm:11.0.1)
│     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│
├─ @labelflow/web@workspace:typescript/web
│  ├─ @labelflow/db@workspace:typescript/db (via workspace:typescript/db)
│  ├─ mjml@npm:4.10.3 (via npm:4.10.3)
│  │  └─ mjml-cli@npm:4.10.3 (via npm:4.10.3)
│  │     └─ chokidar@npm:3.5.2 (via npm:^3.0.0)
│  ├─ release-it@npm:14.11.5 (via npm:14.11.5)
│  │  └─ globby@npm:11.0.4 (via npm:11.0.4)
│  │     └─ fast-glob@npm:3.2.5 (via npm:^3.1.1)
│  ├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber [08ba8] (via workspace:typescript/react-openlayers-fiber [08ba8])
│  ├─ @storybook/react@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│  │  ├─ @storybook/core@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │  └─ @storybook/core-server@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│  │  │     ├─ cpy@npm:8.1.2 (via npm:^8.1.1)
│  │  │     ├─ globby@npm:11.0.3 (via npm:^11.0.2)
│  │  │     ├─ @storybook/builder-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│  │  │     │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     │  │  ├─ glob-base@npm:0.3.0 (via npm:^0.3.0)
│  │  │     │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
│  │  │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     │  ├─ react-dev-utils@npm:11.0.4 [17c6b] (via npm:^11.0.3 [17c6b])
│  │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     ├─ @storybook/manager-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│  │  │     │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  ├─ react-dev-utils@npm:11.0.4 [57ca0] (via npm:^11.0.3 [57ca0])
│  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  ├─ apollo@npm:2.33.9 [08ba8] (via npm:2.33.9 [08ba8])
│  │  ├─ @oclif/config@npm:1.17.0 (via npm:1.17.0)
│  │  │  └─ globby@npm:11.0.3 (via npm:^11.0.1)
│  │  ├─ @oclif/plugin-autocomplete@npm:0.3.0 (via npm:0.3.0)
│  │  │  ├─ @oclif/config@npm:1.18.2 (via npm:^1.13.0)
│  │  │  │  └─ globby@npm:11.0.3 (via npm:^11.0.1)
│  │  │  ├─ cli-ux@npm:5.6.7 (via npm:^5.2.1)
│  │  │  │  └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  │  │  │     ├─ @oclif/config@npm:1.18.2 (via npm:^1.18.2)
│  │  │  │     └─ @oclif/help@npm:1.0.1 (via npm:^1.0.1)
│  │  │  │        └─ @oclif/config@npm:1.18.2 (via npm:1.18.2)
│  │  │  └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  │  ├─ @oclif/plugin-help@npm:3.2.3 (via npm:3.2.3)
│  │  │  ├─ @oclif/config@npm:1.18.2 (via npm:^1.15.1)
│  │  │  └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  │  ├─ @oclif/plugin-not-found@npm:1.2.4 (via npm:1.2.4)
│  │  │  └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  │  ├─ @oclif/plugin-plugins@npm:1.10.1 (via npm:1.10.1)
│  │  │  ├─ cli-ux@npm:5.6.7 (via npm:^5.2.1)
│  │  │  └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  │  ├─ @oclif/plugin-warn-if-update-available@npm:1.7.0 (via npm:1.7.0)
│  │  │  ├─ @oclif/config@npm:1.18.2 (via npm:^1.12.8)
│  │  │  └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  │  ├─ cli-ux@npm:5.6.3 (via npm:5.6.3)
│  │  │  └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  │  └─ @oclif/command@npm:1.8.0 [20d49] (via npm:1.8.0 [20d49])
│  │     └─ @oclif/plugin-help@npm:3.3.1 (via npm:^3)
│  │        ├─ @oclif/config@npm:1.18.2 (via npm:1.18.2)
│  │        ├─ @oclif/help@npm:1.0.1 (via npm:^1.0.1)
│  │        └─ @oclif/command@npm:1.8.16 [df361] (via npm:^1.8.15 [df361])
│  ├─ graphiql@npm:1.5.16 [08ba8] (via npm:1.5.16 [08ba8])
│  │  ├─ codemirror-graphql@npm:1.2.11 [57211] (via npm:^1.2.11 [57211])
│  │  │  └─ graphql-language-service@npm:4.1.4 [57211] (via npm:^4.1.4 [57211])
│  │  │     ├─ graphql-language-service-interface@npm:2.10.2 [079e8] (via npm:^2.10.2 [079e8])
│  │  │     │  ├─ graphql-config@npm:4.1.0 [241ad] (via npm:^4.1.0 [241ad])
│  │  │     │  │  ├─ @graphql-tools/graphql-file-loader@npm:7.3.3 [14d95] (via npm:^7.3.2 [14d95])
│  │  │     │  │  │  └─ globby@npm:11.0.4 (via npm:^11.0.3)
│  │  │     │  │  └─ @graphql-tools/json-file-loader@npm:7.3.3 [14d95] (via npm:^7.3.2 [14d95])
│  │  │     │  │     └─ globby@npm:11.0.4 (via npm:^11.0.3)
│  │  │     │  ├─ graphql-language-service-parser@npm:1.10.4 [079e8] (via npm:^1.10.4 [079e8])
│  │  │     │  │  └─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     │  │     └─ graphql-config@npm:4.1.0 [241ad] (via npm:^4.1.0 [241ad])
│  │  │     │  ├─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     │  └─ graphql-language-service-utils@npm:2.7.1 [079e8] (via npm:^2.7.1 [079e8])
│  │  │     │     └─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     ├─ graphql-language-service-parser@npm:1.10.4 [079e8] (via npm:^1.10.4 [079e8])
│  │  │     ├─ graphql-language-service-types@npm:1.8.7 [079e8] (via npm:^1.8.7 [079e8])
│  │  │     └─ graphql-language-service-utils@npm:2.7.1 [079e8] (via npm:^2.7.1 [079e8])
│  │  └─ graphql-language-service@npm:4.1.4 [57211] (via npm:^4.1.4 [57211])
│  └─ next@npm:12.0.7 [08ba8] (via npm:12.0.7 [08ba8])
│     └─ chokidar@npm:3.5.1 (via npm:3.5.1)
│
└─ labelflow@workspace:.
   ├─ eslint@npm:7.26.0 (via npm:7.26.0)
   │  └─ glob-parent@npm:5.1.2 (via npm:^5.0.0)
   ├─ release-it@npm:14.11.5 (via npm:14.11.5)
   ├─ @graphql-codegen/cli@npm:1.21.4 [3f179] (via npm:1.21.4 [3f179])
   │  ├─ chokidar@npm:3.5.1 (via npm:^3.4.3)
   │  ├─ @graphql-tools/load@npm:6.2.8 [8b930] (via npm:^6 [8b930])
   │  └─ graphql-config@npm:3.3.0 [8b930] (via npm:^3.2.0 [8b930])
   │     └─ @graphql-tools/load@npm:6.2.8 [8b930] (via npm:^6 [8b930])
   ├─ @storybook/builder-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  ├─ @storybook/core-common@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   │  │  ├─ glob-base@npm:0.3.0 (via npm:^0.3.0)
   │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
   │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [e3246] (via npm:^6.0.4 [e3246])
   │  │  └─ chokidar@npm:3.5.1 (via npm:^3.4.2)
   │  └─ react-dev-utils@npm:11.0.4 [e3246] (via npm:^11.0.3 [e3246])
   │     └─ globby@npm:11.0.1 (via npm:11.0.1)
   ├─ @storybook/manager-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  └─ @storybook/core-common@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   ├─ @storybook/react@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
   │  ├─ @storybook/core@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │  └─ @storybook/core-server@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
   │  │     ├─ cpy@npm:8.1.2 (via npm:^8.1.1)
   │  │     ├─ globby@npm:11.0.3 (via npm:^11.0.2)
   │  │     ├─ @storybook/builder-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
   │  │     │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     │  │  ├─ glob-base@npm:0.3.0 (via npm:^0.3.0)
   │  │     │  │  ├─ fork-ts-checker-webpack-plugin@npm:6.2.9 [aeb7b] (via npm:^6.0.4 [aeb7b])
   │  │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     │  ├─ react-dev-utils@npm:11.0.4 [17c6b] (via npm:^11.0.3 [17c6b])
   │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     ├─ @storybook/manager-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
   │  │     │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  ├─ react-dev-utils@npm:11.0.4 [57ca0] (via npm:^11.0.3 [57ca0])
   │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   ├─ @typescript-eslint/eslint-plugin@npm:4.23.0 [3f179] (via npm:4.23.0 [3f179])
   │  └─ @typescript-eslint/experimental-utils@npm:4.23.0 [df25c] (via npm:4.23.0 [df25c])
   │     └─ @typescript-eslint/typescript-estree@npm:4.23.0 [f18cb] (via npm:4.23.0 [f18cb])
   │        └─ globby@npm:11.0.3 (via npm:^11.0.1)
   ├─ @typescript-eslint/parser@npm:4.23.0 [3f179] (via npm:4.23.0 [3f179])
   │  └─ @typescript-eslint/typescript-estree@npm:4.23.0 [792ef] (via npm:4.23.0 [792ef])
   │     └─ globby@npm:11.0.3 (via npm:^11.0.1)
   └─ eslint-config-airbnb-typescript@npm:12.3.1 [3f179] (via npm:12.3.1 [3f179])
      └─ @typescript-eslint/parser@npm:4.23.0 [3f33b] (via npm:^4.4.1 [3f33b])
         └─ @typescript-eslint/typescript-estree@npm:4.23.0 [f18cb] (via npm:4.23.0 [f18cb])
------------------------------------------ trim -------------------------------------------
├─ @labelflow/web@workspace:typescript/web
│  └─ @storybook/react@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│     └─ @storybook/core@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│        └─ @storybook/core-server@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│           └─ @storybook/csf-tools@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│              └─ @mdx-js/mdx@npm:1.6.22 (via npm:^1.6.22)
│                 ├─ remark-mdx@npm:1.6.22 (via npm:1.6.22)
│                 │  └─ remark-parse@npm:8.0.3 (via npm:8.0.3)
│                 │     └─ trim@npm:0.0.1 (via npm:0.0.1)
│                 └─ remark-parse@npm:8.0.3 (via npm:8.0.3)
│
└─ labelflow@workspace:.
   └─ @storybook/react@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
      └─ @storybook/core@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
         └─ @storybook/core-server@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
            └─ @storybook/csf-tools@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
               └─ @mdx-js/mdx@npm:1.6.22 (via npm:^1.6.22)
------------------------------------------ elliptic -------------------------------------------
├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber
│  └─ @storybook/react@npm:6.2.9 [546c7] (via npm:6.2.9 [546c7])
│     ├─ @storybook/core@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │  └─ @storybook/core-server@npm:6.2.9 [cdace] (via npm:6.2.9 [cdace])
│     │     ├─ @storybook/builder-webpack4@npm:6.2.9 [d14e6] (via npm:6.2.9 [d14e6])
│     │     │  ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     │     │  │     └─ node-libs-browser@npm:2.2.1 (via npm:^2.2.1)
│     │     │  │        └─ crypto-browserify@npm:3.12.0 (via npm:^3.11.0)
│     │     │  │           ├─ browserify-sign@npm:4.2.1 (via npm:^4.0.0)
│     │     │  │           │  └─ elliptic@npm:6.5.4 (via npm:^6.5.3)
│     │     │  │           └─ create-ecdh@npm:4.0.4 (via npm:^4.0.0)
│     │     │  │              └─ elliptic@npm:6.5.4 (via npm:^6.5.3)
│     │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     │     ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│     ├─ @storybook/core-common@npm:6.2.9 [57def] (via npm:6.2.9 [57def])
│     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│
├─ @labelflow/web@workspace:typescript/web
│  ├─ browser-builtins@npm:3.3.1 (via npm:3.3.1)
│  │  └─ crypto-browserify@npm:3.9.14 (via npm:~3.9.10)
│  │     ├─ browserify-sign@npm:3.0.8 (via npm:^3.0.1)
│  │     │  └─ elliptic@npm:3.1.0 (via npm:^3.0.0)
│  │     └─ create-ecdh@npm:2.0.2 (via npm:^2.0.0)
│  │        └─ elliptic@npm:3.1.0 (via npm:^3.0.0)
│  ├─ @labelflow/react-openlayers-fiber@workspace:typescript/react-openlayers-fiber [08ba8] (via workspace:typescript/react-openlayers-fiber [08ba8])
│  ├─ @storybook/react@npm:6.3.12 [08ba8] (via npm:6.3.12 [08ba8])
│  │  ├─ @storybook/core@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │  └─ @storybook/core-server@npm:6.3.12 [5fb18] (via npm:6.3.12 [5fb18])
│  │  │     ├─ @storybook/builder-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│  │  │     │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     ├─ @storybook/manager-webpack4@npm:6.3.12 [6ac5e] (via npm:6.3.12 [6ac5e])
│  │  │     │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  │  ├─ @storybook/core-common@npm:6.3.12 [e4560] (via npm:6.3.12 [e4560])
│  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
│  ├─ next@npm:12.0.7 [08ba8] (via npm:12.0.7 [08ba8])
│  │  └─ crypto-browserify@npm:3.12.0 (via npm:3.12.0)
│  └─ node-polyfill-webpack-plugin@npm:1.1.2 [08ba8] (via npm:1.1.2 [08ba8])
│     └─ crypto-browserify@npm:3.12.0 (via npm:^3.12.0)
│
└─ labelflow@workspace:.
   ├─ @storybook/builder-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  └─ @storybook/core-common@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   ├─ @storybook/manager-webpack5@npm:6.3.0-beta.15 [3f179] (via npm:next [3f179])
   │  └─ @storybook/core-common@npm:6.3.0-beta.15 [e3246] (via npm:6.3.0-beta.15 [e3246])
   ├─ @storybook/react@npm:6.3.12 [3f179] (via npm:6.3.12 [3f179])
   │  ├─ @storybook/core@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │  └─ @storybook/core-server@npm:6.3.12 [c4cd3] (via npm:6.3.12 [c4cd3])
   │  │     ├─ @storybook/builder-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
   │  │     │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     │  │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     ├─ @storybook/manager-webpack4@npm:6.3.12 [bff0f] (via npm:6.3.12 [bff0f])
   │  │     │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  │     │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  │     └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   │  ├─ @storybook/core-common@npm:6.3.12 [57ca0] (via npm:6.3.12 [57ca0])
   │  └─ webpack@npm:4.46.0 [aeb7b] (via npm:4 [aeb7b])
   └─ node-polyfill-webpack-plugin@npm:1.1.2 [3f179] (via npm:1.1.2 [3f179])
      └─ crypto-browserify@npm:3.12.0 (via npm:^3.12.0)
```

</details>

Reply:
I would advocate against this suggestion for 2 reasons:

- That is not the point of this PR. Its only purpose was to fix vulnerabilities listed as `Critical` or `High`, not update every package using them as it will require much more time.

- Furthermore, updating top level packages will not necessarily solve the problem. Using the last version of a package does not mean (at all) that every dependencies it uses will be up-to-date. An example from your `why.log`: we use isomorphic-fetch in its 3.0.0 version which is the last one available. This package requires node-fetch in its version 2.6.1 which is vulnerable. So you cannot update the top level package and its dependency remains vulnerable. Using the resolution method as I did prevents from these cases in which you use the last version of a package which uses a vulnerable version of another one.

Reply:
> That is not the point of this PR. Its only purpose was to fix vulnerabilities listed as Critical or High.

That's not a reason for adding some technical debt.

> not update every package using them as it will require much more time

That's your own opinion but it won't take necessarily much more time.
And overall, prioritizing the upgrade of top-level packages instead of their dependees:
* minimizes the number of dependencies we have to take care
* prevents the risks of incompatibilities between our dependencies and their dependees
* reduces the amount of efforts needed overall as we should update more often them anyway

> Furthermore, updating top level packages will not necessarily solve the problem. Using the last version of a package does not mean (at all) that every dependencies it uses will be up-to-date. An example from your why.log: we use isomorphic-fetch in its 3.0.0 version which is the last one available. This package requires node-fetch in its version 2.6.1 which is vulnerable. So you cannot update the top level package and its dependency remains vulnerable. Using the resolution method as I did prevents from these cases in which you use the last version of a package which uses a vulnerable version of another one.

It does not always solve the problem but for most cases it should be sufficient. You'll often find outdated upstreams in your dependency tree but it should be considered as an edge-case, not the norm.  
And it can be a good reason to open a PR on the upstream repo to help them 😉

...or reconsider using them if it takes too long to be accepted, as for isomorphic-fetch: https://github.com/matthew-andrews/isomorphic-fetch/pull/204

Since the library is only used for the backend, I'll take care to make the NestJS version use Axios (_included in NestJS core_), node-fetch or an alternative instead. The library will then disappear by itself once we'll drop `common-resolvers` and `db` 🧹

Reply:
> That's not a reason for adding some technical debt.

10 lines of code that fixes critical vulnerabilities are considered technical debt now ? 🤨 

> * minimizes the number of dependencies we have to take care

No. Setting a version in the `resolutions` key will force every package in our repo to use this version, so you don't have to think `for this package, is that dependency up-to-date ?`. It does not change the number of dependencies, it's the same.

You don't manage the dependencies by themselves, you just force the version to be used across the project.

> * prevents the risks of incompatibilities between our dependencies and their dependees

Yes that's the main caveat, but I have tested every resolution I have integrated.

> * reduces the amount of efforts needed overall as we should update them more often anyway

Sure, but it's not the point of this PR.

>You'll often find outdated upstreams in your dependency tree but it should be considered as an edge-case, not the norm. And it can be a good reason to open a PR on the upstream repo to help them 😉 ...or reconsider using them if it takes too long to be accepted, as for isomorphic-fetch: [matthew-andrews/isomorphic-fetch#204](https://github.com/matthew-andrews/isomorphic-fetch/pull/204)

That would be ideal, but again, it would take much more time. And it's not an opinion, making a PR in an external package to update a dependency cannot be faster than using the resolutions as I did. I think you misunderstood what I am trying to do with this PR, I'm trying to patch security vulns we currently have in production as fast as possible, not having the most up-to-date version of every dependency used by everyone in the npm environment.

I'm closing this PR as it's going nowhere , I will let you open an issue for updating every dependency of the repo.